spk/ps3netsrv/src/wizard/install_uifile_rus

```diff
@@ -0,0 +1,19 @@
+[{
+    "step_title": "Директория с файлами PS3",
```

Comment:
Why not... I should add french translation too then ;)

Reply:
French translation already added )

Reply:
I don’t know this is the right place to ask, if not then I’m sorry about it. Is it possible to give in some more path locations in the ps3netsrv.spk package. In webman you can give in more nethost locations based on Ports 38008, 38009 and so on. On my synology NAS I also have more path’s for PS3 based stuf. Important stuf is on the raid drives les important stuf on a single drive in the system. I have tried to start a second ps3netsrv by command line (SSH) but the applicaties returns with a “error in initialization”

Reply:
@PaddyNL At the moment it is impossible. I will try to add this in the future

Reply:
Thx for the quick reply. Then we will have to wait.